mmcls/models/tta/score_tta.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import List
+
+from mmengine.model import BaseTTAModel
+from mmengine.structures import BaseDataElement
+
+from mmcls.registry import MODELS
+from mmcls.structures import ClsDataSample
+
+
+@MODELS.register_module()
+class AverageScoreTTAModel(BaseTTAModel):
```

Comment:
I think it makes more sense to call it AverageTTAClassifier, at least the search cannot use this.

Reply:
Ok, I have renamed it to `AverageClsScoreTTA`.